buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -103,6 +105,35 @@ class PerformanceTestPlugin : Plugin<Project> {
         tasks.register("performanceReport")
     }
 
+    private
+    fun Project.registerForkPointDistributionTask() {
+        val determineForkPoint = tasks.register("determineForkPoint", DetermineForkPoint::class)
+        val buildForkPointDistribution = tasks.register("buildForkPointDistribution", BuildForkPointDistribution::class) {
+            dependsOn(determineForkPoint)
+        }
+
+        buildForkPointDistribution.get().forkPointDistributionVersion.set(determineForkPoint.get().forkPointDistributionVersion)
+
+        tasks.register("configurePerformanceTestBaseline") {
+            dependsOn(buildForkPointDistribution)
+            doLast {
+                val commitBaseline = determineForkPoint.get().forkPointDistributionVersion.get()
+                project.tasks.withType(DistributedPerformanceTest::class) {
+                    if (baselines?.isEmpty() == true || baselines == "defaults") {
```

Comment:
Why don't you use this:
```
                    if (baselines.isNullOrEmpty() || baselines == "defaults") {
                        baselines = commitBaseline
                    }
```

`baselines` is of type `String?`, and `isNullOrEmpty` works on `String?` and yields a `Boolean` (non-null).